ballerina/channel_csv_io.bal

```diff
@@ -60,35 +60,64 @@ isolated function channelWriteCsv(string path, FileWriteOption option, string[][
     return;
 }
 
-isolated function channelWriteCsvFromStream(WritableChannel writableChannel, stream<string[]|map<anydata>, Error?> csvStream) returns
+isolated function channelWriteCsvFromStream(string path, FileWriteOption option, stream<string[]|map<anydata>, Error?> csvStreamToWrite) returns
 Error? {
-    WritableCSVChannel csvChannel = check getWritableCSVChannel(writableChannel);
-    do {
-        if csvStream is stream<string[], Error?> {
-            record {|string[] value;|}|Error? csvRecordString = csvStream.next();
+    string[] headersFromCSV = [];
+    string[] headersFromStream = [];
+    boolean skipHeaders = false;
+    if csvStreamToWrite is stream<string[], Error?> {
+        WritableCSVChannel csvChannel = check getWritableCSVChannel(check openWritableCsvFile(path, option = option));
+        do {
+            record {|string[] value;|}|Error? csvRecordString = csvStreamToWrite.next();
```

Comment:
are we handling the error return here?

Reply:
Same question. From the logic seems its ignored. Is it the correct way?